storage/innobase/xtrabackup/doc/source/advanced/page_tracking.rst

```diff
@@ -0,0 +1,126 @@
+
+1. Install the mysqlbackup component and enable it on the server: 
+
+  .. code-block:: bash
+
+     INSTALL COMPONENT "file://component_mysqlbackup";
+
+2. Check whether the mysqlbackup component is installed succesfuly:
+
+  .. code-block:: bash
+
+     SELECT COUNT(1) FROM mysql.component WHERE component_urn='file://component_mysqlbackup';
+
+.. note::
+
+   The mysqlbackup component processes all the tracked pages in memory. If the component does not have enough available memory to process all the tracked pages, the component throws an error and exits. For example, if an incremental backup uses 200GB, the component can additionally use about 100MB of memory to store the page tracking data.
+
+Using page tracking
+-------------------
+
+If the server page flushing rate is low or it uses a big buffer pool, you can start page tracking before creating the first backup. Thus, you ensure that page tracking log sequence number (LSN) is more than the checkpoint LSN of the server.
+
+For the very first full backup using page tracking, PXB may have a delay. The following is an example of the message you can receive:
+   
+  .. code-block:: bash
+
+     xtrabackup: pagetracking: Sleeping for 1 second, waiting for checkpoint lsn 17852922 to reach to page tracking start lsn 21353759
+   
+This delay ensures that the page tracking start LSN is more than the backup end checkpoint LSN which is required for the next incremental backup.
+
+.. note::
+
+   To use the page tracking options, you must have the `Backup_ADMIN <https://docs.oracle.com/cd/E17952_01/mysql-enterprise-backup-8.0-en/mysqlbackup.privileges.html>`__ privileges.
+
+Start page tracking
+-------------------
+
+After the mysqlbackup component is loaded and active on the server, start the page tracking:
+
+  .. code-block:: bash
+
+     SELECT mysqlbackup_page_track_set(true);
+
+When you start page tracking, it creates a file under the server's datadir to collect data about changed pages. This file grows until you stop the page tracking. If you stop the server and then restart it, page tracking creates a new file but also keeps the old one. The old file continues to grow until you stop the page tracking explicitly.
+
+To purge old page tracking data that you no longer need, do the following steps:
+
+  .. code-block:: bash
+
+     SELECT mysqlbackup_page_track_set(false);
+     SELECT mysqlbackup_page_track_purge_up_to(9223372036854775807);
+     /* Specify the LSN up to which you want to purge page tracking data. 9223372036854775807 is the highest possible LSN which purges all page tracking files.*/
+     SELECT mysqlbackup_page_track_set(true);
+
+Enable page tracking
+--------------------
+
+Enable the page tracking functionality for the full and incremental backups with the ``--page-tracking`` option.
+
+An example of enabling full backup.
+
+  .. code-block:: bash
+
+     xtrabackup --backup --target-dir=$FULL_BACK --page-tracking
+
+An example of enabling incremental backup.
```

Comment:
replace enabling with how to take incremental backup using page-tracking

Reply:
Updated